daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -2699,6 +2691,12 @@ func NewDeviceSet(root string, doInit bool, options []string, uidMaps, gidMaps [
 				return nil, err
 			}
 			devices.xfsNospaceRetries = val
+		case "dm.log_level":
```

Comment:
I'd prefer to not expose this to users. Passing through the logrus logging level seems the easiest way to set this.